common/mixnet-contract/src/mixnode.rs

```diff
@@ -35,6 +40,101 @@ pub enum Layer {
     Three = 3,
 }
 
+#[derive(Debug, Clone, JsonSchema, PartialEq, Serialize, Deserialize, Copy)]
+pub struct NodeRewardParams {
+    period_reward_pool: Uint128,
+    k: Uint128,
+    total_epoch_uptime: Uint128,
+    #[serde(default = "current_block_height")]
```

Comment:
FIY, I've just removed this function, as we probably only need it for migrations.